### PR TITLE
enable travis

### PR DESCRIPTION
**Pull request by dshimo**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM websphere-liberty:microProfile
+FROM websphere-liberty:18.0.0.3-webProfile7
```
Comment (yeekangc) [inline Dockerfile]: The POM references 18.0.0.4 while the Dockerfile is 18.0.0.3?

Comment (dshimo) [reply Dockerfile]: Fixed

File: src/main/liberty/config/server.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <server description="Sample Liberty server">
 
     <featureManager>
-        <feature>microprofile-1.3</feature>
+        <feature>microProfile-1.3</feature>
```
Comment (yeekangc) [inline src/main/liberty/config/server.xml:4]: Should this be 1.4?

**Review by yeekangc — APPROVED** [review]
Aside from the latest comment re 18.0.0.4 for the server.xml, the PR looks good to me.

File: src/main/liberty/config/server.xml (hunk 2 above)
Comment (dshimo) [reply src/main/liberty/config/server.xml:4]: an issue to be marked

**Pull request merged**